systemtest/src/test/java/io/strimzi/systemtest/operators/FeatureGatesIsolatedST.java

```diff
@@ -330,4 +333,72 @@ void testSwitchingStrimziPodSetFeatureGateOnAndOff(ExtensionContext extensionCon
 
         ClientUtils.waitForClientsSuccess(producerName, consumerName, INFRA_NAMESPACE, messageCount);
     }
+
+    /**
+     * UseKRaft feature gate
+     */
+    @IsolatedTest("Feature Gates test for enabled UseKRaft gate")
+    @Tag(INTERNAL_CLIENTS_USED)
+    public void testKRaftMode(ExtensionContext extensionContext) {
+        assumeFalse(Environment.isOlmInstall() || Environment.isHelmInstall());
+
+        final String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
+        final String producerName = "producer-test-" + new Random().nextInt(Integer.MAX_VALUE);
+        final String consumerName = "consumer-test-" + new Random().nextInt(Integer.MAX_VALUE);
+        final String topicName = KafkaTopicUtils.generateRandomNameOfTopic();
+
+        final LabelSelector zkSelector = KafkaResource.getLabelSelector(clusterName, KafkaResources.zookeeperStatefulSetName(clusterName));
+        final LabelSelector kafkaSelector = KafkaResource.getLabelSelector(clusterName, KafkaResources.kafkaStatefulSetName(clusterName));
```

Comment:
You can use the `TestStorage` here and then just use the getters.

Reply:
Why would one do it when we have a public API with it?

Reply:
The `TestStorage` contains all variables that you are using in the test - such as `clusterName`, `producerName`, `zkSelector` etc. Then you can do just `testStorage.getProducerName()`, `testStorage.getKafkaSelector()` and you don't have to create all kind of variables by yourself. Or what do you mean?

Reply:
So, how do you get the TestStorage instance? TBH, the test is more or less copy paste frm the other tests. None of the other tests in this class seems to use the TestStorage.

Reply:
Yeah I know that the test is copy paste more or less :D 
`final TestStorage testStorage = new TestStorage(extensionContext, Constants.INFRA_NAMESPACE);`

Reply:
I don't think it makes sense to create an instance of some class just to get some value I can get more easily with existing static methods. It does not seem to have any benefits.

Reply:
We are using it in lot of tests, it saves few lines of code and also removes need of creating lot of variables, which is on one place - in the `TestStorage`. FMPOV it's better to have struct which contains everything than using the static methods repeatedly for each test.

Reply:
Ok, maybe it should be then fixed separately for all tests in the class.